mods/ts/sequences/civilian.yaml

```diff
@@ -862,6 +862,27 @@ gakodk:
 	critical-idle:
 		Start: 2
 		ShadowStart: 5
+	large-lights: gakodk_a
+		Length: 12
+		Tick: 200
+	damaged-large-lights: gakodk_a
+		Start: 12
+		Length: 12
+		Tick: 200
+	small-light: gakodk_b
+		Length: 12
```

Comment:
This had `Length: 22` in TS. https://github.com/LipkeGu/TSIniFiles/blob/master/art.ini#L4438